### PR TITLE
Fixed formatting and added links to relevant pages

### PR DESCRIPTION
**Pull request by northyg**

### CODE REVIEW

**Review by unknwon — APPROVED** [review]

**Comment by unknwon** [comment]
Thanks!

File: doc/admin/privileges.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 Site administrators are able to impersonate regular users in two ways:
-1. View and update the settings of any user.
+
+1. [View and update the settings of any user.](https://docs.sourcegraph.com/admin/config/settings#editing-global-settings-for-site-admins)
 2. Initiate GraphQL API calls on behalf of any other user.
 
 ## Receive site alerts
 
-Site administrators see update notifications and other site-level alerts (visible as a banner across the top of the screen) that may be invisible to non-admin users.
+Site administrators see update notifications and other [site-level alerts](https://docs.sourcegraph.com/admin/observability/alerting#understanding-alerts) (visible as a banner across the top of the screen) that may be invisible to non-admin users.
```
Comment (bobheadxi) [inline doc/admin/privileges.md:22]: Note that site level alerts are not the observability alerts linked here, they are e.g. license expiry and update banner

File: doc/admin/privileges.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
 # Site administrator privileges
 
-Site administrators have full administrative access to the Sourcegraph instance. In many cases, they also control the deployment environment. Special privileges are granted to site-admin users.
+Site administrators have [full administrative access to the Sourcegraph instance.](https://docs.sourcegraph.com/admin/config) In many cases, they also control the [deployment environment.](https://docs.sourcegraph.com/admin/install) Special privileges are granted to site-admin users.
```
Comment (bobheadxi) [inline doc/admin/privileges.md:3]: Relative links instead so we can get link checks?

**Review by bobheadxi — COMMENTED** [review]
nice!

File: doc/admin/privileges.md (hunk 2 above)
Comment (northyg) [reply doc/admin/privileges.md:3]: @bobheadxi Oh! That was an oversight. I'll fix them 😄

**Pull request closed without merging**